modules/account-lib/src/coin/dot/transactionBuilder.ts

```diff
@@ -372,6 +388,20 @@ export abstract class TransactionBuilder extends BaseTransactionBuilder {
 
   // endregion
 
+  /**
+   * Adds a signature to the transaction.
+   *
+   * @param publicKey public key that produced the signature
+   * @param signature raw signature as a hex encoded Buffer
+   */
+  addSignature(publicKey: BasePublicKey, signature: Buffer): void {
+    this._signatures.push({ publicKey, signature });
+  }
+
+  get signatures(): Interface.Signature[] {
```

Comment:
ditto, if it isn't in the interface already

Reply:
realized this was not needed. we were already using `this._signatures` to access this like the other properties in this file.

Removed the method